Sources/VideoPlayerView.swift

```diff
@@ -92,31 +92,35 @@ public final class VideoPlayerView: _PlatformBaseView {
         player?.status == .readyToPlay &&
         isLooping
     }
-    
+
     private func registerNotifications() {
         NotificationCenter.default
             .addObserver(self,
                          selector: #selector(registerNotification(_:)),
                          name: .AVPlayerItemDidPlayToEndTime,
                          object: player?.currentItem)
+        #if os(iOS) || os(tvOS)
```

Comment:
According to https://developer.apple.com/documentation/uikit/uiapplication/1622944-willenterforegroundnotification

`UIApplication.willEnterForegroundNotification` seems only works on iOS (macCatalyst) and tvOS.